### PR TITLE
[G2M] Added Additional unit tests to data table and widget for validity

### PR DESCRIPTION
**Pull request by gulsahsarsilmaz**
Added additional prop testing for Data Table and Widget Trend.

### CODE REVIEW

File: __tests__/data-table/data-table.js [hunk 1]
```diff
@@ -17,4 +17,30 @@ describe('<DataTable/>', () => {
     const tree = component.toJSON()
     expect(tree).toMatchSnapshot()
   })
+
+  // Failing Cases
```
Comment (woozyking) [inline __tests__/data-table/data-table.js:21]: why are these considered failing cases?

Comment (gulsahsarsilmaz) [reply __tests__/data-table/data-table.js:21]:
```
There are only two hard things in Computer Science: cache invalidation and naming things.
Phil Karlton
```
😆 I couldnt name them properly.

Comment (woozyking) [reply __tests__/data-table/data-table.js:32]: perhaps "defaultProps" cases?

Comment (gulsahsarsilmaz) [reply __tests__/data-table/data-table.js:32]: - [ ] Update comment on first Default Props - Missing Cases
- [ ] Update comment on second Default Props - Valid Cases

**Pull request merged**